rpcs3/rpcs3qt/debugger_frame.cpp

```diff
@@ -1156,6 +1121,20 @@ void debugger_frame::DoStep(bool step_over)
 			m_debugger_list->ShowAddress(cpu->get_pc() + 4, false);
 		}
 
+		if (step_over && cpu->id_type() == 0x55)
+		{
+			const bool was_paused = cpu->is_paused();
+			static_cast<rsx::thread*>(cpu)->pause_on_draw = true;
+
+			if (was_paused)
+			{
+				Q_EMIT RunBtnPress();
```

Comment:
you only emit signals.
slots can be called like any function

Reply:
Fixed.